src/index.tsx

```diff
@@ -26,6 +26,7 @@ export {
     useSetOverscrollColor,
     OverscrollColorProvider,
 } from './overscroll-color-context';
+export {default as OverMediaContextProvider, useOverMediaContext} from './over-media-context';
```

Comment:
## Remark

If we want this approach to be the final one (i.e. forget about the idea of a new variant for ThemeVariant), we can export this outside of Mistica. Otherwise, I would not export it, because it will cause breaking changes when we remove it.

WDYT?